apps/notifications/src/panel/indices-to-html/index.js

```diff
@@ -108,6 +109,27 @@ function render_range( new_sub_text, new_sub_range, range_info, range_data, opti
 			}
 			build_chunks( new_sub_text, new_sub_range, range_data, new_container, options );
 			break;
+		case 'figure':
+			if ( range_info.hasOwnProperty( 'class' ) && range_info.class.includes( 'wp-block-table' ) ) {
+				new_container = document.createElement( 'a' );
+				new_container.setAttribute( 'href', options.sourceUrl );
+				new_container.setAttribute( 'target', '_blank' );
+				new_container.setAttribute( 'rel', 'noopener noreferrer' );
+				new_classes.push( 'table-fallback-link' );
+				new_container.innerHTML = i18n.translate( 'Click here to view table.', {
+					comment: 'Fallback text when viewing a table in a subscribed post notification',
+				} );
+			} else {
+				new_container = document.createElement( range_info_type );
+				if ( range_info.hasOwnProperty( 'class' ) ) {
+					new_classes.push( range_info.class );
+				}
+				if ( range_info.hasOwnProperty( 'style' ) ) {
+					new_container.setAttribute( 'style', range_info.style );
+				}
+				build_chunks( new_sub_text, new_sub_range, range_data, new_container, options );
+			}
+			break;
```

Comment:
This should be in our `gutenberg-block-rendering-fallbacks` plugin where we have other fallbacks for Core blocks (see paYKcK-Iu-p2):

```php
function table_block_fallback( $block_content, $block ) {
  $context = Display_Context\get_current_context();

  // Only run for notifications.
  if ( Display_Context\NOTIFICATIONS !== $context ) {
    return $block_content;
  }

  // Only run for table blocks.
  if ( $block['blockName'] !== 'core/table' ) {
    return $block_content;
  }

  return '<a href="' . get_permalink() . '" style="display:block; margin-bottom: 15px;" target="_blank" rel="noopener noreferrer">' . __( 'Click here to view table.' ) . '</a>';
}
add_filter( 'render_block', __NAMESPACE__ . '\table_block_fallback', 10, 2 );
```

Reply:
Aha. That would simplify things. Cool. :) Will update both PRs accordingly.

Reply:
Gah, good catch @mmtr 🙃